ember-addon-main.js

```diff
@@ -109,12 +187,24 @@ module.exports = {
       pluginCacheKey: pluginInfo.cacheKeys
     };
 
+    if (this.legacyPluginRegistrationCacheBustingRequired !== false) {
```

Comment:
the `!== false` here makes it hard for my brain to read this

Reply:
kk, ya I can fix that up